src/zip.h

```diff
@@ -426,15 +429,6 @@ extern ZIP_EXPORT int zip_extract(const char *zipname, const char *dir,
                                                           void *arg),
                                   void *arg);
 
-/**
- * Sets global CRC-32 checksum function.
- *
- * @param crc32_func crc32 function (init value, buffer, buffer size)
- */
-extern ZIP_EXPORT void
-zip_crc32_func(unsigned long (*crc32_func)(unsigned long crc, const void *buf,
```

Comment:
Would be good to have this functionality (for performance reasons).

Reply:
...someone asked for this functionality so I've added. It's  never been part of miniz.

Reply:
Did you patch it into miniz directly?

Reply:
yes, our miniz, contains many _patches_/additions